apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1/SubscriberClient.cs

```diff
@@ -1500,9 +1499,21 @@ private void HandleAckResponseForExactlyOnce(Task writeTask, List<string> ackIds
                 StartPush();
                 return;
 
-                // All methods below are local methods applicable to exactly once subscription only.
-                // They are not used in non-exactly once subscription.
-                
+                // All methods below are local methods applicable to exactly once delivery subscriptions only.
```

Comment:
People don't usually go up a few blocks to read block comments like this one. Best to repeat the info succinctly in the comments for each method.

Reply:
Sure Prad. Will address this in the next commit.